gdal/swig/python/scripts/gdal2tiles.py

```diff
@@ -651,8 +651,15 @@ def __init__(self, arguments ):
         # KML generation
         self.kml = self.options.kml
 
-        # Output the results
+        # LC_CTYPE check
+        if 'UTF-8' not in os.environ.get("LC_CTYPE", ""):
```

Comment:
ok I have it. I'll add a couple of tests and I'll submit an updated PR.

Cheers